src/observers/http1/http1_observer.c

```diff
@@ -55,6 +57,10 @@ static int rx_request(qd_http1_decoder_connection_t *hconn, const char *method,
     hreq->vflow = vflow_start_record(VFLOW_RECORD_BIFLOW_APP, th->vflow);
     vflow_set_string(hreq->vflow, VFLOW_ATTRIBUTE_PROTOCOL, version_minor == 1 ? "HTTP/1.1" : "HTTP/1.0");
     vflow_set_string(hreq->vflow, VFLOW_ATTRIBUTE_METHOD, method);
+    vflow_set_uint64(hreq->vflow, VFLOW_ATTRIBUTE_OCTETS, 0);
+    vflow_set_uint64(hreq->vflow, VFLOW_ATTRIBUTE_OCTETS_REVERSE, 0);
+    vflow_add_rate(hreq->vflow, VFLOW_ATTRIBUTE_OCTETS, VFLOW_ATTRIBUTE_OCTET_RATE);
+    vflow_add_rate(hreq->vflow, VFLOW_ATTRIBUTE_OCTETS_REVERSE, VFLOW_ATTRIBUTE_OCTET_RATE_REVERSE);
```

Comment:
I did not add the VFLOW_ATTRIBUTE_OCTET_RATE_REVERSE to the http2 observer. I will create a PR for that.